lib/components/Datepicker/Datepicker.tsx

```diff
@@ -83,13 +98,6 @@ export const Datepicker = React.forwardRef<HTMLInputElement, DatepickerProps>(fu
             name={name}
             placeholder={placeholder}
             className="ds-w-full ds-rounded-l ds-border-y ds-border-l ds-border-border-gray ds-bg-white ds-p-5 ds-font-arial ds-text-black ds-outline-none placeholder:ds-text-inactive-gray"
-            onInput={(e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Hare is the redundant onInput handler. It didn't seem to do much except cause troubles.